modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc

```diff
@@ -57,6 +57,20 @@ For all available Cypher types, see the section on xref::values-and-types/proper
 | Feature
 | Details
 
+a|
+label:functionality[]
+label:new[]
+label:beta[]
+
+[source, cypher, role=noheader]
+----
+CALL cdc.current()
+CALL cdc.earliest()
+CALL cdc.query(from, selectors)
+----
+
+| Introduction of the change data capture feature. For details see link:{neo4j-docs-base-uri}/cdc/{page-version}/introduction[Change Data Capture]
```

Comment:
It is always capitalised when Ive read about it in internal release notes and elsewhere.

```suggestion
| Introduction of the Change Data Capture (CDC) feature.
For details see link:{neo4j-docs-base-uri}/cdc/{page-version}/introduction[Change Data Capture]
```